monochart/templates/_common_metadata_labels.tpl

```diff
@@ -38,7 +38,7 @@ Example output:
 Kubernetes standard labels
 */}}
 {{- define "common.labels.standard" -}}
-app.kubernetes.io/name: {{ include "common.name" . }}
```

Comment:
I think for `app.kubernetes.io/instance` we want to stay with the `common.name`. For my test app on kOps Staging, this would be `monochart-testing` instead of `monochart-testing-staging`. The instance should be the release name.

Reply:
Hmm, is it really the case? I might not understand how `common.name` is defined exactly, but seems like for kops services this label gets `spoton-monochart` value everywhere?

Reply:
Yes sorry, I looked at `common.name` wrong. Agreed that it should not continue to be used. The `.Release.Name` is also a problem in that (at least on kOps clusters) it is include `-staging` or `-prod`, which we don't want. I believe what we want is `common.fullname`, which is used for the [deployment's name](https://github.com/SpotOnInc/helmcharts/blob/master/monochart/templates/deployment.yaml#L6). That should give us the service name without `-staging` or `-prod`.

Reply:
That seems right, changed.